libraries/AP_UAVCAN/dsdl/ardupilot/gnss/20002.Heading.uavcan

```diff
@@ -0,0 +1,5 @@
+bool heading_valid
+bool heading_accuracy_valid
+
+float32 heading
```

Comment:
it would be nice to fit in one can pkt, if we use int16 in centi-degrees (or float16) then we can make it fit I think